openjdk/build.xml

```diff
@@ -81,7 +81,15 @@
 							<property name="jdkName" value="openjdk-aarch64-jdk8u" />
   						</then>		
 					<else>
-						<property name="jdkName" value="openjdk-jdk${JDK_VERSION}u" />
+						<if>
+							<equals arg1="${JDK_VERSION}" arg2="14" />
```

Comment:
see my note in the issue, can you try using one of the extra conditions provided by ant-contrib http://ant-contrib.sourceforge.net/tasks/tasks/more_conditions.html

would suggest this:
`<property name="jdkName" value="openjdk-jdk${JDK_VERSION}" />
<if>
 	<isLessThan arg1="${JDK_VERSION}" arg2="14" />
<then>
 	<property name="jdkName" value="openjdk-jdk${JDK_VERSION}u" />
 </then>
</if>`

Reply:
Ah cool I didn't see this bit of documentation, it isn't in he standard antlib one

Reply:
Also, do any of the other build.xml files use the openjdk (hs) repos? I couldn't see any that did but worth asking

Reply:
Correct, the other types of tests we run come from other repos.  build.xml file in the openjdk directory is the only place where we pull material from that openjdk-jdk${JDK_VERSION} repo.

Reply:
From what I can tell, `islessthan` is not going to be helpful here. 
From: http://ant-contrib.sourceforge.net/tasks/tasks/more_conditions.html
> These conditions are suitable for use in the <bool> element. Unfortunately, they cannot be used in the <condition> task`

Reply:
If that is the case, then I will ask you to switch your solution around a bit to explicitly check if equals 8 through 13 append u, else do not append u so we do not have to come back in 15+ to update.